src/middleware/packages/ldp/package.json

```diff
@@ -8,7 +8,10 @@
   },
   "dependencies": {
     "jsonld": "^1.8.1",
+    "moleculer": "^0.13.11",
+    "moleculer-web": "^0.9.0-beta7",
```

Comment:
Tu es sûr de devoir mettre `moleculer-web` en dépendance ?
Je vois que le package `moleculer` est utilisé pour les Errors, mais je vois pas pour `moleculer-web`

Reply:
c'était le cas dans des itérations précédentes Errors de 'moleculer-web'. A nettoyer.